pandas/core/generic.py

```diff
@@ -7611,6 +7611,16 @@ def replace(
             )
 
         inplace = validate_bool_kwarg(inplace, "inplace")
+        if inplace:
+            if not PYPY and using_copy_on_write():
+                refcount = 2 if PY311 else 3
```

Comment:
This can be updated now the other PR is merged

Reply:
Done